### ISSUE TITLE
Twig Syntax Error

### ISSUE THREAD

**Issue by Visiluna** [nystudio107/craft-templatecomments#2]
Template Comments 1.0.2
Craft 3.0.28

After installing _Template Comments_ from the plugin store, copying the `config.php` file to `craft/config`, and renaming it to `templatecomments.php`, I am getting the following error when loading any front-end pages.

```
Twig Syntax Error – Twig_Error_Syntax
An exception has been thrown during the compilation of a template ("Node "body" does not exist for Node "nystudio107\templatecomments\web\twig\nodes\CommentsNode".").
↵
Caused by: LogicException
Node "body" does not exist for Node "nystudio107\templatecomments\web\twig\nodes\CommentsNode".
in /Users/philip/Google Drive/Projects/TripMoxie/craft/vendor/twig/twig/lib/Twig/Node.php at line 144
```

Control panel still works fine.

**Comment by khalwat** [comment]
Interesting. What do your templates look like? Is it possible you have some entirely empty templates (nothing in them at all)?

**Comment by Visiluna** [comment]
I went back through all my templates and none are empty.

**Comment by khalwat** [comment]
So I'm at a loss here. Can you somehow get to me your project files so I can replicate it here? Or could we do a screen sharing at some point to have a look at what is going on?

**Comment by Visiluna** [comment]
Sure. Let me see what I can put together.

**Comment by Visiluna** [comment]
Due to project deadlines, I kind of dropped the ball on this.

I now have an archive of the project I can send you, but I prefer not to post that publicly. Let me know where I can send it.

Thanks.

**Comment by wihodges** [comment]
Just had this same problem. Was editing _entry template for a section, inserted a form tag and input with no markup or styling, an since I was using browsersync, it all the sudden threw the error. When I deleted what I just did, error persists. Deleted the whole template except extends layout and block content with "hello world" and error still doesn't work. Interestingly all the other pages still render. 
Disabled the plugin Templates Comments and all works again.

Sounds like witchCraft, but thats my best recount of what happened?  

Not sure if this helps but these were the first couple of screenshots of the error:

<img width="1245" alt="screenshot 2018-12-02 21 55 42" src="https://user-images.githubusercontent.com/286956/49352360-29f3f980-f67d-11e8-9da9-699b2705b167.png">

<img width="1234" alt="screenshot 2018-12-02 21 57 33" src="https://user-images.githubusercontent.com/286956/49352390-4abc4f00-f67d-11e8-873a-4b3a27aba8b9.png">

**Comment by khalwat** [comment]
@wihodges If you clear your compiled template caches, does it then work again?

**Comment by khalwat** [comment]
@Visiluna yeah if you don't mind, that'd be great -- andrew@nystudio107.com -- try the above tho? Clear all of your caches and see if the issue persists?

**Comment by Visiluna** [comment]
@khalwat Clearing caches did not fix the problem. I cleared them all, not just compiled templates.

**Comment by wihodges** [comment]
Ok tried clearing cache - still errors.

**Comment by khalwat** [comment]
Fixed in 1.0.3 -> https://github.com/nystudio107/craft-templatecomments/releases/tag/1.0.3

Thanks for sending along the project @Visiluna -- that allowed me to reproduce it, and then fix it.
